### DISCUSSION TITLE
Can NOT run prisma for a month, Windows 11, VSCode

### DISCUSSION THREAD

**Discussion by hiennqtaki** [prisma/prisma#23838] [Q&A]
### Bug description
```
PS C:\GitHubCenter\gamifa_backend> npx prisma generate
Environment variables loaded from .env
Prisma schema loaded from prisma\schema.prisma
Error: request to https://downloads.prisma.sh/all_commits/473ed3124229e22d881cb7addf559799debae1ab/windows/query_engine.dll.node.gz.sha256 failed, reason: write EPROTO D8220000:error:0A00010B:SSL routines:ssl3_get_record:wrong version number:c:\ws\deps\openssl\openssl\ssl\record\ssl3_record.c:355:

```
--------------
No Python, no Openssl, everything is working until i go to toilet!

### Prisma information

```prisma
// Add your schema.prisma
```

```ts
// Add your code using Prisma Client
```

### Environment & setup

- OS: 
- Database: 
- Node.js version: 

### Prisma Version

```
prisma                  : 5.12.1
@prisma/client          : 5.12.1
Computed binaryTarget   : windows
Operating System        : win32
Architecture            : x64
Node.js                 : v20.10.0
Query Engine (Node-API) : libquery-engine 473ed3124229e22d881cb7addf559799debae1ab (at node_modules\@prisma\engines\query_engine-windows.dll.node)
Schema Engine           : schema-engine-cli 473ed3124229e22d881cb7addf559799debae1ab (at node_modules\@prisma\engines\schema-engine-windows.exe)
Schema Wasm             : @prisma/prisma-schema-wasm 5.12.0-21.473ed3124229e22d881cb7addf559799debae1ab
Default Engines Hash    : 473ed3124229e22d881cb7addf559799debae1ab
Studio                  : 0.499.0
```

**Comment by ludralph** [comment]
Hi @hiennqtaki,

Thank you for raising this question.

This may be a Network problem on your machine. Can you download `https://downloads.prisma.sh/all_commits/473ed3124229e22d881cb7addf559799debae1ab/windows/query_engine.dll.node.gz.sha256` manually in your browser?

**Reply by hiennqtaki** [reply]
yes i can, but many my projects work perfect, just one project has a problem, and it stop me from do anything...

**Comment by hiennqtaki** [comment]
by the way, this is my package.json:
`
{
  "name": "",
  "version": "1.3.9",
  "description": "",
  "author": "",
  "private": true,
  "type": "commonjs",
  "license": "UNLICENSED",
  "scripts": {
    "prebuild": "rimraf dist",
    "build": "nest build  --webpack --webpackPath webpack-hmr.config.js",
    "build:webpack": "rimraf dist && webpack --env production --mode=production --config webpack.config.js",
    "format": "prettier --write \"src/**/*.ts\" \"test/**/*.ts\"",
    "start": "nest start",
    "start:debug": "nest start --debug --watch",
    "start:dev": "rimraf dist && nest start --watch",
    "start:prod": "node dist/main",
    "lint": "eslint \"{src,apps,libs,test}/**/*.ts\" --fix",
    "test": "node ./test.mjs"
  },
  "dependencies": {
    "@nestjs/common": "^10.3.7",
    "@nestjs/config": "^3.2.2",
    "@nestjs/core": "^10.3.7",
    "@nestjs/mapped-types": "^2.0.5",
    "@nestjs/microservices": "^10.3.7",
    "@nestjs/mongoose": "^10.0.5",
    "@nestjs/platform-express": "^10.3.7",
    "@nestjs/platform-socket.io": "^10.3.7",
    "@nestjs/schedule": "^4.0.1",
    "@nestjs/typeorm": "^10.0.2",
    "@nestjs/websockets": "^10.3.7",
    "@prisma/client": "^5.12.1",
    "@types/cookie-parser": "^1.4.7",
    "@types/multer": "^1.4.11",
    "@types/node-cron": "^3.0.9",
    "@types/request": "^2.48.10",
    "aws-sdk": "^2.1598.0",
    "axios": "^1.6.8",
    "bluebird": "^3.7.2",
    "class-transformer": "^0.5.1",
    "class-validator": "^0.14.1",
    "cookie-parser": "^1.4.6",
    "crypto": "^1.0.1",
    "date-and-time": "^3.1.1",
    "express": "^4.18.3",
    "file-type": "19.0.0",
    "firebase": "^10.11.0",
    "firebase-functions": "^4.9.0",
    "google-play-billing-validator": "^2.1.3",
    "googleapis": "^134.0.0",
    "https-proxy-agent": "^7.0.4",
    "jsonwebtoken": "^9.0.2",
    "lodash": "^4.17.21",
    "mongoose": "^8.3.1",
    "mysql2": "^3.9.4",
    "nestjs-redis": "^1.3.3",
    "node-cron": "^3.0.2",
    "nodemailer": "^6.9.12",
    "oauth-1.0a": "^2.2.6",
    "passport": "^0.7.0",
    "passport-google-oauth": "^2.0.0",
    "passport-google-oauth20": "^2.0.0",
    "paypal-rest-sdk": "^1.8.1",
    "prisma": "^5.12.1",
    "qs": "^6.12.0",
    "request": "^2.88.2",
    "rimraf": "^5.0.5",
    "rxjs": "^7.8.1",
    "strip-ansi": "^7.1.0",
    "stripe": "^15.1.0",
    "tls": "^0.0.1",
    "typeorm": "^0.3.20"
  },
  "devDependencies": {
    "@nestjs/bull": "^10.1.0",
    "@nestjs/cli": "^10.3.2",
    "@nestjs/schematics": "^10.1.1",
    "@nestjs/testing": "^10.3.7",
    "@types/babel__core": "^7.20.5",
    "@types/bcrypt": "^5.0.2",
    "@types/express": "^4.17.13",
    "@types/jsonwebtoken": "^9.0.6",
    "@types/lodash": "^4.17.0",
    "@types/node": "^20.12.7",
    "@types/nodemailer": "^6.4.14",
    "@types/passport": "^1.0.16",
    "@types/passport-google-oauth": "^1.0.42",
    "@types/passport-google-oauth20": "^2.0.11",
    "@types/paypal-rest-sdk": "^1.7.9",
    "@types/webpack-node-externals": "^3.0.4",
    "@typescript-eslint/eslint-plugin": "^7.6.0",
    "@typescript-eslint/parser": "^7.6.0",
    "bcrypt": "^5.1.1",
    "bull": "^4.12.2",
    "eslint": "^9.0.0",
    "eslint-config-prettier": "^9.1.0",
    "eslint-plugin-prettier": "^5.1.3",
    "prettier": "^3.2.5",
    "raw-loader": "^4.0.2",
    "run-script-webpack-plugin": "^0.2.0",
    "ts-loader": "^9.5.1",
    "ts-node": "^10.9.2",
    "typescript": "^5.4.5",
    "webpack": "^5.90.3",
    "webpack-cli": "^5.1.4",
    "webpack-node-externals": "^3.0.0"
  }
}

`

**Reply by ludralph** [reply]
Hi @hiennqtaki 

Has this issue been resolved? You may need to disable your antivirus if any on your windows computer to see if it resolves this issue.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!;
